packages/kbn-build/src/commands/clean.ts

```diff
@@ -28,7 +33,8 @@ export async function run(projects, projectGraph, { rootPath }) {
 
     for (const dir of directoriesToDelete) {
       const deleting = del(dir, { force: true });
-      ora.promise(deleting, relative(rootPath, dir));
+      // @types/ora doesn't expose `ora.promise`.
```

Comment:
:( Maybe PR a fix to DefinitelyTyped?

Reply:
Yes, the plan is to augment/workaround everything we need, merge (if everything is fine) and then contribute to DefinitelyTyped and get rid of these in our codebase. The reason: PRs to DT may take some time. What do you think?

Reply:
Works for me :+1:

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/23699

Reply:
🎉

Reply:
:heavy_check_mark: